src/modules/user-twofa/user-twofa.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, HttpCode } from '@nestjs/common';
+import { UserTwoFaService } from './user-twofa.service';
+import { ApiOkResponse } from '@nestjs/swagger';
+
+@Controller('two-factor-auth')
+export class UserTwoFaController {
+  constructor(private readonly twoFactorAuthService: UserTwoFaService) {}
+
+  @Get('generate-qr')
+  @HttpCode(200)
+  @ApiOkResponse({ description: 'token' })
+  async generateQr() {
+    try {
```

Comment:
try catch chỗ này không có ý nghĩa gì
nó tương đương với `return await this.twoFactorAuthService.generateQr();`